src/main/java/com/sollace/coppergolem/entity/CopperGolemEntity.java

```diff
@@ -120,9 +118,7 @@ protected void initDataTracker() {
     }
 
     public BlockInteraction getFinder(int maxDistance) {
-        return finders.computeIfAbsent(Registry.ITEM.getId(getStackInHand(Hand.MAIN_HAND).getItem()), id -> {
-            return BlockInteraction.create(this, maxDistance);
-        });
+        return finders.computeIfAbsent(Registry.ITEM.getId(getStackInHand(Hand.MAIN_HAND).getItem()), id -> BlockInteraction.create(this, maxDistance));
```

Comment:
Lambda formatting should not be changed from what they were before. I've used them with blocks in some places for readability purposes as otherwise the lines get too long to read.